10326.xml

```diff
@@ -0,0 +1,724 @@
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>Robot PM</Name>
+    <Description1><![CDATA[This LWM2M Object includes PM(counter and gauge) supported by all robots.]]></Description1>
+    <ObjectID>10326</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10326</ObjectURN>
+    <LWM2MVersion>1.0</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Single</MultipleInstances>
+    <Mandatory>Mandatory</Mandatory>
+    <Resources>
+	  <Item ID="51">
+        <Name>Battery Level</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
+        <Description><![CDATA[Referrence to a PM item, similar with Corelnk in LwM2M 1.1, 
+			&lt;/object ID/object instance ID/resoure ID&gt;, for example:&lt;/10315/0/2&gt;]]></Description>
+      </Item>
+
+      <Item ID="300">
+        <Name>Battery Temperature</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>°C</Units>
+        <Description><![CDATA[Battery Temperature.]]></Description>
+      </Item>	  
+	  <Item ID="301">
+        <Name>Temperature</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>°C</Units>
+        <Description><![CDATA[Environment Temperature.]]></Description>
+      </Item>
+      <Item ID="302">
+        <Name>Humidity</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
+        <Description><![CDATA[Environment Humidity.]]></Description>
+      </Item>
+      <Item ID="303">
+        <Name>PM2.5</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>μg/m3</Units>
```

Comment:
Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

Reply:
μg/m3 is the unit of PM2.5, the concentration of an air pollutant is given in micrograms (one-millionth of a gram) per cubic meter air.

Reply:
replace μg/m3 with ug/m3 which is micrograms per cubic meter

Reply:
Replaced, thanks.